tests/ClientTest.php

```diff
@@ -204,4 +214,55 @@ public function testCreateResponseWithHeadersEqualToEmptyArray()
         $this->assertInstanceOf(HeaderCollection::className(), $headersCollection);
         $this->assertEquals([], $headersCollection->toArray());
     }
+
+    public function testCreateRequestShortcut()
+    {
+        $method = 'POST';
+        $url = 'url';
+        $data = ['data'];
+        $headers = ['headers'];
+        $options = ['options'];
+
+        $client = new Client();
+        /** @var Request $request */
+        $request = $this->invoke($client, 'createRequestShortcut', [$method, $url, $data, $headers, $options]);
```

Comment:
No need to do if we'll leave it as public...

Reply:
You mean private?

Reply:
https://github.com/yiisoft/yii2-httpclient/pull/187#discussion_r279402801

Reply:
I'm not sure I understand what you mean. `createRequestShortcut` was private and I changed it to protected to be able to invoke it in the test. Why is that considered backward compatibility break?

Reply:
I misread it yesterday. It's alright. Sorry.